docs/management/admin/host-isolation-exceptions.asciidoc

```diff
@@ -0,0 +1,41 @@
+[[host-isolation-exceptions]]
+[chapter]
+= Host isolation exceptions
+
+You can configure host isolation exceptions for specific IP addresses that <<host-isolation-ov, isolated hosts>> are still allowed to communicate with, even when blocked from the rest of your network. Isolated hosts can still send data to {es} and {kib}, so you don't need to set up host isolation exceptions for them.
+
+Host isolation exceptions support IPv4 addresses, with optional CIDR (classless inter-domain routing) notation.
```

Comment:
```suggestion
Host isolation exceptions support IPv4 addresses, with optional classless inter-domain routing (CIDR) notation.
```

Reply:
Recommending this switch because I've generally seen the definition first and the acronym second.